h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
A question is whether we need this information at all. Shouldn't we be able to tell, at the completion callback, which way we've scrolled by which edge the element is aligned with?

Also, we have not patched jschannel. As far as I know we have not patched any vendor libs. This is not a practice I have ever tolerated.